plugin/src/main/java/io/fabric8/maven/plugin/mojo/infra/AbstractInstallMojo.java

```diff
@@ -282,16 +309,16 @@ protected Platform getPlatform() {
         }
     }
 
-    protected void runGofabric8(File command, String ... args) throws MojoExecutionException {
+    protected void executeCommand(File command, String binName, String ... args) throws MojoExecutionException {
         // Be sure to run in batch mode
         List<String> argList = new ArrayList<>(Arrays.asList(args));
-        argList.add("--batch");
+        //argList.add("--batch");
```

Comment:
Yes, but I see that the `--batch` parameter has not been added to calls done in child classes of `AbstractInstallMojo` (`ClusterStartMojo`, `ClusterStopMojo` etc.)...